FragmentNavigation/src/main/java/com/infomaniak/core/fragmentnavigation/FragmentNavigationExt.kt

```diff
@@ -32,6 +33,10 @@ fun Fragment.isAtInitialDestination(substituteClassName: String? = null): Boolea
     return findNavController().isAtInitialDestination(allowedInitialClassName = substituteClassName ?: javaClass.name)
 }
 
+fun Activity.isAtInitialDestination(navController: NavController, substituteClassName: String? = null): Boolean {
+    return navController.isAtInitialDestination(allowedInitialClassName = substituteClassName ?: javaClass.name)
+}
+
```

Comment:
Does it really need to be public or is it only used inside `Activity.safelyNavigate()`?

Reply:
It used to need it, but not anymore !
Nice catch.